### PR TITLE
sd : update the script to take only 2 partitions

### PR DESCRIPTION
**Pull request by nadenislamarre**
need to be updated to make more tests, but works.
make to boot a bit bigger in case of update.

Signed-off-by: Nicolas Adenis-Lamarre nicolas.adenis-lamarre@gmail.com

### CODE REVIEW

File: scripts/linux/mk-sdcard-img.sh [hunk 1]
```diff
@@ -88,8 +85,7 @@ for image in ${images}; do
     echo "* Cleaning ${image}..."
     umount "${SDCARD_BOOT_PWD}" > /dev/null || true
     umount "${SDCARD_ROOT_PWD}" > /dev/null || true
-    umount "${SDCARD_SHARE_PWD}" > /dev/null || true
-    kpartx -sd "${image}" > /dev/null || true
+    kpartx -sd "${image}" > /dev/null
```
Comment (pasnox) [inline scripts/linux/mk-sdcard-img.sh:88]: Why removing dev null here? Sometime the img exists but is not mounted at all

File: scripts/linux/mk-sdcard-img.sh [hunk 2]
```diff
@@ -88,8 +85,7 @@ for image in ${images}; do
...
     rm -f "${image}" > /dev/null
```
Comment (pasnox) [inline scripts/linux/mk-sdcard-img.sh:89]: I mean || true

**Comment by nadenislamarre** [comment]
Hi,

i'll do some other changes on the script, but it works already very well ;-)
I suggest i finish and we discuss on irc about the changes ?

File: scripts/linux/mk-sdcard-img.sh (hunk 1 above)
Comment (pasnox) [inline scripts/linux/mk-sdcard-img.sh:88]: I mean || true

File: scripts/linux/mk-sdcard-img.sh [hunk 3]
```diff
@@ -156,68 +142,43 @@ do
...
-fi
-releaseUser="`basename \"${releaseHome}\"`"
-releaseFilePath="${releaseHome}/public_html/${baseName}.tar.xz"
-mkdir -p "`dirname \"${releaseFilePath}\"`"
-chown -R ${releaseUser}:www-data "${SDCARD_IMG_FILE_PATH}"
-chmod -R u+rwx,g+rwx,o+rx "${SDCARD_IMG_FILE_PATH}"
-tar -cJvf "${releaseFilePath}" -C "`dirname \"${SDCARD_IMG_FILE_PATH}\"`" "`basename \"${SDCARD_IMG_FILE_PATH}\"`" > /dev/null
-chown -R ${releaseUser}:www-data "${releaseFilePath}"
-chmod -R u+rwx,g+rwx,o+rx "${releaseFilePath}"
-rm "${SDCARD_IMG_FILE_PATH}"
```
Comment (pasnox) [inline scripts/linux/mk-sdcard-img.sh:220]: Move that to an unused function instead of removing it - i do want to use it on my server

**Comment by pasnox** [comment]
Ok. ;)

**Comment by nadenislamarre** [comment]
the "Moving to testing area..." seemed to be too specific to your needs (and not related to the sd card creating).
but we can move it to an other script calling the sd card creating and then moving the img to your www.
Please note that /home/recalbox, public_html, www-data are all specifics and probably unwanted for most people. This part is very specific to your needs i guess for the way you deliver the sd card.
However, very good job. It should be used for nightly + official builds in my opinion.

**Comment by digitalLumberjack** [comment]
It will be used in nightly buids soon, but i agree with you the Moving is specific to the build environnement, i will remove that and use it in a specific script on my server.

**Comment by nadenislamarre** [comment]
I will add options in the script,
like the sd card file path, the sd compression and the help.

**Comment by nadenislamarre** [comment]
Are we sure we need openjdk-8-jdk ?

**Pull request merged**